eve/main.yml

```diff
@@ -39,3 +32,22 @@ stages:
             npm run flow &&
             npm run lint
           haltOnFailure: false
+  post-merge:
+    worker:
+      type: kube_pod
+      path: eve/workers/pod-unit-tests/pod.yaml
+      images:
+        docker-unit-tests:
+          context: '.'
+          dockerfile: eve/workers/pod-unit-tests/Dockerfile
```

Comment:
Could you explain to me why the context needs to be `.` because I took a look at the dockerfile, and it doesn't seem to be needed. You could even replace those lines to :
```yaml
docker-unit-tests: 'eve/workers/pod-unit-tests'
```

and same story for the one on the `pre-merge` story as it's the same worker.